src/components/BTable/BTable.md

```diff
@@ -0,0 +1,68 @@
+The headers array holds objects that represent columns
+
+| Key      | Required/Optional | Default | Description                              |
+| -------- | ----------------- | ------- | ---------------------------------------- |
+| name     | Required          |         | This is what will be displayed as the column header.  |
+| key      | Required          |         | This is the matching key for that column to access the corresponding data in the rows. |
+| sort     | Optional          |         | Enables sorting on that column.                   |
+| render   | Optional          |         | Holds the rendering function name if applicabled.                   |
+
+```js
+
+const makeLink = function(data) {
+  return `<a href=${ data.id }>${ data.name }</a>`
+}
+
+const makeImage = function(data) {
+  return `<img src="${ data.source }" alt="${ data.name }" width="30" height="30">`
+}
+
+const makeLinkGroup = function(data) {
+  let html = ''
+  data.forEach(function(action) {
+    switch(action) {
+      case 'delete':
+        html += `<span class="ion-ios-trash" style="color: #4d90fe; font-size: 18px"></span><a href=${ data.id } class='link-group'> Delete</a>`
```

Comment:
maybe change these to `alert` or something so that it doesn't 404 when someone clicks...

or use the popup?

Reply:
the popup didnt' exist when I started building this.  I will integrate it though when its done